Makefile

```diff
@@ -111,7 +112,7 @@ check_shell:
 sanity: vet fix lint fmt
 
 cover:
```

Comment:
a feature request (not sure how hard it is)
``` sh
make cover [package=package_name]
```
Behavior: if package is set (for example, `package=ledger`) then COVERPKG_PACKAGES would be ledger + all subpackages.

Benefit:
1. partial run for only interested packages
2. faster run - can be done on dev machine

Reply:
OK — I added support to do `make cover PACKAGE=ledger`